### PR TITLE
Coverity: Initialize class members 

### PR DESCRIPTION
**Pull request by ssakash**
CID numbers are provided in the commit messages.

### CODE REVIEW

**Comment by ramapcsx2** [comment]
Looking at these, it seems wondrous that anything worked. Good job :)

**Comment by refractionpcsx2** [comment]
> stream = NULL;

I wouldn't quite call that initializing something, the whole point is to set the variables to something, NULL is not something :P

**Comment by gregory38** [comment]
(didn't check the context) but it is still better to have NULL than a random address, isn't it?

File: pcsx2/CDVD/InputIsoFile.cpp [hunk 1]
```diff
@@ -160,6 +160,10 @@ int InputIsoFile::FinishRead3(u8* dst, uint mode)
 InputIsoFile::InputIsoFile()
 {
 	_init();
+	m_reader = NULL;
+	ReadUnit = 0;
+	m_current_count = 0;
```
Comment (gregory38) [inline pcsx2/CDVD/InputIsoFile.cpp]: m_current_count  is never used. Better remove it.

File: pcsx2/CDVD/InputIsoFile.cpp [hunk 2]
```diff
@@ -160,6 +160,10 @@ int InputIsoFile::FinishRead3(u8* dst, uint mode)
 InputIsoFile::InputIsoFile()
 {
 	_init();
+	m_reader = NULL;
```
Comment (gregory38) [inline pcsx2/CDVD/InputIsoFile.cpp]: m_reader could be done in the init phase? In this case we could remove the one in the destructor.

To be investigated for m_current_count and ReadUnit.

File: pcsx2/DebugTools/Debug.h [hunk 3]
```diff
@@ -73,7 +73,7 @@ class SysTraceLog : public TextFileTraceLog
 
 	// Pass me a NULL and you *will* suffer!  Muahahaha.
 	SysTraceLog( const SysTraceLogDescriptor* desc )
-		: TextFileTraceLog( &desc->base ) {}
+		: TextFileTraceLog( &desc->base ), PrePrefix(NULL) {}
```
Comment (gregory38) [inline pcsx2/DebugTools/Debug.h]: I don't think PrePrefix is used anywhere. Might be a better idea to remove it with the SetPrefix function.

**Comment by refractionpcsx2** [comment]
> (didn't check the context) but it is still better to have NULL than a random address, isn't it?

Maybe, unless it decides "NULL" is just whatever it feels like :P I guess it depends if Coverity is happy with it.

**Comment by ssakash** [comment]
updated the code, Thanks to @gregory38 for the suggestions.

**Comment by gregory38** [comment]
Ok great. We are good. By the way, no need to send me a done for each change. Just send useful information. A final summary is enough.

**Pull request merged**